src/components/Courses/component/Registration/Registration.tsx

```diff
@@ -21,7 +21,7 @@ const defaultFormFields = {
 	email: 'test@example.com',
 	password: '123123',
 };
-const notify = (message) => toast(message);
+const notify = (message: string) => toast(message);
 
 const Registration = () => {
 	const [formFields, setFormFields] = useState(defaultFormFields);
```

Comment:
no type

Reply:
I moved toast to a separate typed helper function.